roles/confluent.kafka/templates/server_kerberos.properties.j2

```diff
@@ -4,8 +4,8 @@
 
 sasl.enabled.mechanisms=GSSAPI
 sasl.mechanism.inter.broker.protocol=GSSAPI
-listeners=SASL_PLAINTEXT://{{inventory_hostname}}:{{broker.config.sasl_port}}
-advertised.listeners=SASL_PLAINTEXT://{{inventory_hostname}}:{{broker.config.sasl_port}}
+listeners=SASL_PLAINTEXT://{{inventory_hostname}}:{{kafka_port}}
```

Comment:
Recommend leaving in 'broker' as kafka is not a component but the platform name.